changelog.md

```diff
@@ -0,0 +1,19 @@
+
+# Change Log
+All notable changes to this project will be documented in this file.
+ 
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+ 
+## [0.1.1] - 2023-03-15
+ 
+This change is a workspace addition of protobufs codegen capabilities. The new package arch3-proto is the core change.
+ 
+### Added
+- [arch3.js-0.1.1](https://github.com/archway-network/arch3.js)
+  arch-proto: Generate and integrate protobuf clients.
```

Comment:
```suggestion
### Added

- *arch-proto*: added package with protobuf clients [#37](https://github.com/archway-network/arch3.js/pull/37)
```

The description should also reference which PR or commit introduced the specific change.